x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/mobile/errors/group_id_samples.spec.ts

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import { timerange } from '@kbn/apm-synthtrace-client';
+import { service } from '@kbn/apm-synthtrace-client/src/lib/apm/service';
+import { orderBy } from 'lodash';
+import type { ApmSynthtraceEsClient } from '@kbn/apm-synthtrace';
+import { APIReturnType } from '@kbn/apm-plugin/public/services/rest/create_call_apm_api';
+import { getErrorGroupingKey } from '@kbn/apm-synthtrace-client/src/lib/apm/instance';
+import type { DeploymentAgnosticFtrProviderContext } from '../../../../../ftr_provider_context';
+import { config, generateData } from './generate_data';
+
+type ErrorGroupSamples =
+  APIReturnType<'GET /internal/apm/services/{serviceName}/errors/{groupId}/samples'>;
+
+type ErrorSampleDetails =
+  APIReturnType<'GET /internal/apm/services/{serviceName}/errors/{groupId}/error/{errorId}'>;
+
+export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderContext) {
+  const apmApiClient = getService('apmApi');
+  const synthtrace = getService('synthtrace');
+
+  const serviceName = 'synth-go';
+  const start = new Date('2021-01-01T00:00:00.000Z').getTime();
+  const end = new Date('2021-01-01T00:15:00.000Z').getTime() - 1;
+
+  async function callErrorGroupSamplesApi({ groupId }: { groupId: string }) {
+    const response = await apmApiClient.readUser({
+      endpoint: 'GET /internal/apm/services/{serviceName}/errors/{groupId}/samples',
+      params: {
+        path: {
+          serviceName,
+          groupId,
+        },
+        query: {
+          start: new Date(start).toISOString(),
+          end: new Date(end).toISOString(),
+          environment: 'ENVIRONMENT_ALL',
+          kuery: '',
+        },
+      },
+    });
+    return response;
+  }
+
+  async function callErrorSampleDetailsApi(errorId: string) {
+    const response = await apmApiClient.readUser({
+      endpoint: 'GET /internal/apm/services/{serviceName}/errors/{groupId}/error/{errorId}',
+      params: {
+        path: {
+          serviceName,
+          groupId: 'foo',
+          errorId,
+        },
+        query: {
+          start: new Date(start).toISOString(),
+          end: new Date(end).toISOString(),
+          environment: 'ENVIRONMENT_ALL',
+          kuery: '',
+        },
+      },
+    });
+    return response;
+  }
+
+  describe('Group id samples', () => {
+    it('handles the empty state', async () => {
+      const response = await callErrorGroupSamplesApi({ groupId: 'foo' });
+      expect(response.status).to.be(200);
+      expect(response.body.occurrencesCount).to.be(0);
+    });
+
+    // FLAKY: https://github.com/elastic/kibana/issues/177654
+    describe.skip('when samples data is loaded', () => {
```

Comment:
**question:** do you think it can be safely migrated as "deployment-agnostic" in `skip` state without being run against MKI and ESS?

DO tests gave pretty strict stability requirements: https://github.com/elastic/kibana/tree/main/x-pack/test/api_integration/deployment_agnostic#tagging-and-skipping-the-tests

I'm afraid that later you might face issues to re-enable due to those Cloud-first testing requirement

Reply:
TBH,  I don't think it's safe. I could try unskip these tests